exercises/chapter4/test/Main.purs

```diff
@@ -122,13 +123,17 @@ This line should have been automatically deleted by resetSolutions.sh. See Chapt
           Assert.equal (sort [ [ 3, 4, 5 ], [ 5, 12, 13 ], [ 6, 8, 10 ] ])
             $ sort
             $ triples 13
-      suite "Exercise - factorize" do
-        test "Test small non-prime number" do
-          Assert.equal [ 3, 2 ]
-            $ factorize 6
-        test "Test number that uses the prime numbers less than 10" do
-          Assert.equal [ 7, 5, 3, 2 ]
-            $ factorize 210
+      suite "Exercise - primeFactors" do
+        let
+          primeFactorsTest :: Int -> Array Int -> _
+          primeFactorsTest n xs =
+            test (show n) do
+              Assert.equal (sort xs)
+                $ sort
+                $ primeFactors n
+        primeFactorsTest 6 [3, 2]
+        primeFactorsTest 18 [3, 3, 2]
+        primeFactorsTest 210 [ 7, 5, 3, 2 ]
```

Comment:
These tests previously depended on the ordering of returned array and would not check for repeated factors.